### ISSUE TITLE
The Keeley Consulting Group: Climate change’s projected effect on NC coast

### ISSUE THREAD

**Issue by kirbyweyand** [kirbyweyand/thekeeleygroup#1]
A new report from the [Coastal Resources Commission’s Science Panel](http://www.yuuby.com/note/12785/the-keeley-consulting-group-climate-change-s-projected-effect-on-nc-coast) on Coastal Hazards (SPCH) reveals that North Carolina coast is gradually getting lapped by the sea.

Naturally, this is not just an issue that’s exclusive to North Carolina as climate change is resulting in stronger typhoons and worse flooding around the world — it is, however, one of the highly-prone locations.

That’s according to the detailed document released by the SPCH based on tidal gauge measurements collected from Southport, Beaufort, Oregon Inlet, Duck and Wilmington. Data from the Intergovernmental Panel on Climate Change also served as their basis for the 43-page report which projects higher sea levels in three decades. It predicts Duck’s sea level will increase by 4-10 inches in 2045, while Southport’s is expected to rise 2-9 inches.

[The Keeley Consulting Group](http://www.thekeeleygroup.com/) considers the report on sea level as a critical one, especially when the coastal and homeowners welfare are taken into account. And according to Professor Stanley Riggs of Department of Geological Sciences (East Carolina University), the numbers may not sound alarming but the effects could be.

“There are two different things: One is the long-term change going on in the globe — slow, systematic. You also have the short-term. The short-term ranges from daily tides to storm surges — 5, 10, 15, 20-foot change in sea levels in hours,” Riggs said.

The bottom line is, as the sea levels gradually increase, short-term effects also worsen. Therefore, he fears that many houses along the coast of North Carolina won’t even reach 30 years — if there’s no federal funding to improve the problematic areas anyway.
